src/dyld/darling

```diff
@@ -66,8 +66,19 @@ setup_prefix() {
     rm -f "$1/dev"
     ln -sf "system-root/dev" "$1/dev"
 
+    rm -rf "$1/sandboxed-home"
+
     rm -f "$1/home"
```

Comment:
I'd say this function (`setup_prefix`) **REQUIRES** a rewrite with lot more checks. For example, for emptyness of `$1` (nobody should **never** do anything like `rm -f ${variable}/<path_on_filesystem>`. It can SUDDENLY appear the case of empty variable in the future, which lead to bad event) and if it is really necessarily to rm'ing that files (or they all already fine).

So, if you really want rm'ing the path prepended by variable, check for variablr